tests/pipeline/hisat2/hisat2_skip_index.nf.test

```diff
@@ -0,0 +1,194 @@
+nextflow_pipeline {
+
+    name "Test Workflow main.nf"
+    script "main.nf"
+    options "-profile test,docker"
+    tag "pipeline"
+
+    test("HISAT2 (Skip Index) (GSE110004)") {
+
+        when {
+            params {
+                outdir       = "$outputDir"
+                aligner      = "hisat2"
+                hisat2_index = false
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(path("$outputDir/pipeline_info/software_versions.yml")).match("software_versions") },
```

Comment:
Snapshot will need to be updated whenever we update module versions, it's an extra task, but that way we probably won't forgot about it in the changelog as well, I like it.

Reply:
yes i have been thinking about that and believe it will be a good check before merges